### DISCUSSION TITLE
spindle and mode pause

### DISCUSSION THREAD

**Discussion by JoseThorel** [vlachoudis/bCNC#1710] [Q&A]
Hi,
in the pause mode, why  the spindle button is disabled?
I would like to be able to stop the spindle for cleaning
(sorry for the bad english)

**Comment by MARIOBASZ** [comment]
Look in grbl for the Door  command.
I think the park position should be set.

**Comment by JoseThorel** [comment]
during the pause on BCNC all commands are disabled

for example with the CANDLE software during the pause, 
the spindle can be stopped by pressing the button on the software control panel
The spindle restarts automatically when the work is restarted
I don't understand why BCNC can't do this
Maybe BCNC is more focused on security .........

**Reply by Harvie** [reply]
it boils down to GRBL design.
imagine you are doing big G02 circle and user pauses in middle of that, also there are several g-code commands already buffered in GRBL. how do you jog somewhere safely? how do you resume?

it would be nice to have, but is not trivial to fix.

personaly i think it would make more sense to use something like klipper instead of grbl, because it does not send g-codes which might make things more straightforward when implementing such thing. but that currently klipper only can drive 3d printers.

**Comment by MARIOBASZ** [comment]
can you write the g code it sends to stop the spindle?
I suspect it sends Hex 0x84
If so, it would be nice to add a button to the panel that remains active with pause
